restapi/introduction.mdx

```diff
@@ -24,8 +24,8 @@ All our endpoints live under the url `https://api.axiom.co` and follow the REST
 Here’s an example in **curl**:
 
 ```bash
-curl -X 'POST' 'https://api.axiom.co/v1/datasets/$DATASET_NAME/ingest' \
-  -H 'Authorization: Bearer $API_TOKEN' \
+curl -X 'POST' 'https://api.axiom.co/v1/datasets/DATASET_NAME/ingest' \
```

Comment:
https://api.axiom.co/v1/datasets/DATASET_NAME/ingest

As discussed in 1:1, let’s adopt 
`https://api.axiom.co/v1/datasets/{id}/ingest` as convention instead since Mintlify does not produce clear demarkation of variables like Google in their style guide